packages/cli-link-assets/src/tools/cleanAssets/android.ts

```diff
@@ -0,0 +1,129 @@
+import fs from 'fs-extra';
+import OpenType from 'opentype.js';
+import path from 'path';
+import {
+  FontFamilyMap,
+  FontXMLObject,
+  REACT_FONT_MANAGER_JAVA_IMPORT,
+  getAddCustomFontMethodCall,
+  getFontFamily,
+  getFontResFolderPath,
+  getProjectFilePath,
+  getXMLFontId,
+  normalizeString,
+  removeLineFromJavaFile,
+  toArrayBuffer,
+  xmlBuilder,
+  xmlParser,
+} from '../helpers/font/androidFontAssetHelpers';
+import {AndroidCleanAssetsOptions, CleanAssets} from './types';
+
+const cleanAssets: CleanAssets = (assetFiles, options) => {
```

Comment:
we could possibly extract some parts of this function and share it with `copyAssets`

Reply:
Will do it after finishing the tests, so it will be safer to refactor 🙂